### DISCUSSION TITLE
npm ERR! code E401 -> Error: Command npm install exited with 1

### DISCUSSION THREAD

**Discussion by BassamHager** [vercel/vercel#9142] [Help] [answered]
The first deploy worked properly, then I added new things pushed then tried the command: `vercel` and got this err below:

`❯ vercel`
`
Vercel CLI 28.10.1
🔍  Inspect: https://vercel.com/bassamhager/bassam-hager-portfolio-nextts/5U1MiCbgQGe7Cdc1JG77qBfbYaWR [2s]
Error: Command "npm install" exited with 1
Error: Check your logs at https://bassam-hager-portfolio-nextts-n4yvykpfl-bassamhager.vercel.app/_logs or run `vercel logs bassam-hager-portfolio-nextts-n4yvykpfl-bassamhager.vercel.app`

![](https://media.discordapp.net/attachments/1057356941770362880/1057356941917175858/image.png)

![](https://cdn.discordapp.com/attachments/1057356941770362880/1057587769658069062/image.png)

[2022-12-28T09_18_53_906Z-debug-0.log](https://github.com/vercel/vercel/files/10313511/2022-12-28T09_18_53_906Z-debug-0.log)

I've tried re-logging (didn't work)

![](https://cdn.discordapp.com/attachments/1057356941770362880/1057591427510571028/image.png)

I've tried [this Stack Overflow page](https://stackoverflow.com/questions/67419637/e401-unable-to-authenticate-your-authentication-token-seems-to-be-invalid) as well, not sure if I did it accurately, but if you confirmed the cause I'll go ham on it.

Thank you very much in advance 🙏

**Comment by amyegan** [comment, marked as answer]
Hi @BassamHager! Are you using any private packages? I ran into this error before when I hadn't set up my project to use private dependencies. This is the guide that helped me: https://vercel.com/guides/using-private-dependencies-with-vercel
